lib/Dialect/FIRRTL/Transforms/IMConstProp.cpp

```diff
@@ -603,6 +596,9 @@ void IMConstPropPass::visitStrictConnect(StrictConnectOp connect) {
       return;
   }
 
+  if (isAggregate(dest.getOwner()))
+    return mergeLatticeValue(connect.dest(), srcValue);
```

Comment:
Should this be `markOverdefined(connect.dest())`?

Reply:
Yea, it is probably simpler to just mark both sides overdefined.